devtools/projects/ng-devtools/src/lib/devtools-tabs/router-tree/router-tree.component.html

```diff
@@ -1,12 +1,133 @@
 <div class="router-tree-wrapper">
-  <div>
-    <input
-      matInput
-      (input)="searchRoutes($event)"
-      placeholder="Search routes"
-      class="filter-input"
-    />
-    <mat-checkbox (change)="togglePathSettings()">Show Full Path </mat-checkbox>
-  </div>
-  <ng-tree-visualizer-host #routerTree />
+  <as-split unit="pixel" direction="vertical" [gutterSize]="9" [disabled]="true">
+    <as-split-area size="50">
+      <div>
+        <input
+          matInput
+          (input)="searchRoutes($event)"
+          placeholder="Search routes"
+          class="filter-input"
+        />
+        <mat-checkbox (change)="togglePathSettings()">Show Full Path </mat-checkbox>
```

Comment:
Typo: Trailing space.

```suggestion
        <mat-checkbox (change)="togglePathSettings()">Show Full Path</mat-checkbox>
```